tests/test_server.py

```diff
@@ -198,9 +238,20 @@ def test_streaming(client):
 
         output = pd.read_csv(f.name).set_index("task_id")
         assert not output["error"].any(), "adala returned errors"
-        assert (
-            output["output"] == expected_output
-        ).all(), "adala did not return expected output"
+
+        # check for expected output
+        expected_outputs = data.set_index("task_id")[output_column].tolist()
+        actual_outputs = [eval(item)[output_column] for item in output.output.tolist()]
+        for actual_output, expected_output in zip(actual_outputs, expected_outputs):
+            if skills[0]["type"] == "EntityExtraction":
+                # Live generations may be flaky, check only 3 entities are presented
+                assert len(actual_output) == len(expected_output)
+                assert actual_output[0]["label"] == expected_output[0]["label"]
```

Comment:
nit: can we just get the lists of labels and do an `assert actual_output == expected_output` to avoid requiring test examples to have 3 items?

Reply:
done